docker-compose.yml

```
@@ -47,6 +47,20 @@ services:
       # - AZURE_API_KEY=<your_azure_api_key>
       # - AZURE_API_BASE=<your_azure_api_base>
       # - AZURE_API_VERSION=<your_azure_api_version>
+      # Amazon Bedrock Support:
+      # Amazon Bedrock is a managed service that enables you to invoke LLMs and bill them through your AWS account.
+      # To use Amazon Bedrock as the LLM provider for Skyvern, specify the following environment variables.
+      # 1. In the AWS IAM console, create a new AWS IAM User (name it whatever you want)
+      # 2. Assign the "AmazonBedrockFullAccess" policy to the user
+      # 3. Generate an IAM Access Key under the IAM User's Security Credentials tab
+      # 4. In the Amazon Bedrock console, go to "Model Access"
+      # 5. Click Modify Model Access button
+      # 6. Enable "Claude 3.5 Sonnet v2" and save changes
+      - ENABLE_BEDROCK=true
```

can you add them commented-out by default? 
```suggestion
      # - ENABLE_BEDROCK=true
```